### DISCUSSION TITLE
Who composed famous classical piece The Four Seasons?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#480] [Galaxy Trivia] [answered]
**Bonus:** When was it written?

**Comment by peterbaccaro** [comment, marked as answer]
The Four Seasons are a group of four violin concerti by Italian composer **Antonio Vivaldi**, each of which gives a musical expression to a season of the year.  They were written roughly between **1720** and **1723** and were published in **1725**.
